code/renderers/vue3/src/render.ts

```diff
@@ -49,12 +81,33 @@ export function renderToDOM(
       Use "() => ({ template: '<my-comp></my-comp>' })" or "() => ({ components: MyComp, template: '<my-comp></my-comp>' })" when defining the story.
       `,
     });
-    return;
+    return () => {
+      teardown(domElementKey);
+    };
   }
 
-  showMain();
+  if (!existingApp || forceRemount) {
+    const reactiveArgs = args ? reactive(args) : args;
```

Comment:
the key here is building `reactive(args)` and passing them as the second parameter (`props`) in `h(element, reactiveArgs)`. This makes it so that any changes are propagated to the component.